homeassistant/components/onvif/camera.py

```diff
@@ -58,6 +58,7 @@
 PTZ_NONE = "NONE"
 
 SERVICE_PTZ = "onvif_ptz"
+SERVICE_REBOOT = "onvif_reboot"
```

Comment:
Please register it as `onvif.reboot` instead of `camera.onvif_reboot`

Reply:
Sure can do, but what about the inconsistency toward ptz .. that command is `camera.onvif_ptz` and not `onif.ptz`

Reply:
You can move that one too and mark it as a breaking change. We're in the process of moving all these services.